tilelang/language/atomic.py

```
@@ -313,7 +313,7 @@ def atomic_addx2(dst: Buffer, value: PrimExpr, return_prev: bool = False) -> Pri
     """
     atomic_addx2_op = op.Op.get("tl.atomic_addx2_elem_op") if return_prev else op.Op.get("tl.atomic_addx2_elem_op")
```

Ruff (0.14.14)

[warning] Useless `if`-`else` condition

(RUF034)